### PR TITLE
[plugin.video.southpark_unofficial] v0.5.2

### PR DESCRIPTION
**Pull request by wargio**
### Description

Added season 23, Python3 support and downgraded LICENSE to GPLv2

### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.southpark_unofficial/southpark.py [hunk 1]
```diff
@@ -3,15 +3,23 @@
...
-import urllib2
 import datetime
 import xbmc
 import xbmcplugin
 import xbmcgui
 import xbmcaddon
 import xml.etree.ElementTree as ET
 
+def is_py3():
+	return sys.version_info[0] > 2
```
Comment (enen92) [inline plugin.video.southpark_unofficial/southpark.py]: you can use `sys.version_info.major`

Comment (enen92) [reply plugin.video.southpark_unofficial/southpark.py]: `PY3 = sys.version_info_major > 2`

File: plugin.video.southpark_unofficial/addon.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="plugin.video.southpark_unofficial" name="South Park" version="0.5.1" provider-name="Deroad">
+<addon id="plugin.video.southpark_unofficial" name="South Park" version="0.5.2" provider-name="Deroad">
```
Comment (enen92) [inline plugin.video.southpark_unofficial/addon.xml]: can you change the license to `GPL-2.0` please (spdx identifier)?

Comment (wargio) [reply plugin.video.southpark_unofficial/addon.xml]: why?

Comment (wargio) [reply plugin.video.southpark_unofficial/addon.xml]: I'll change the license to GPL2

**Pull request merged**